mk/spksrc.wheel.mk

```diff
@@ -59,7 +59,7 @@ pre_wheel_target: wheel_msg_target
 
 build_wheel_target: $(PRE_WHEEL_TARGET)
 	@if [ ! -z "$(WHEELS)" ] ; then \
-		$(foreach e,$(shell cat $(WORK_DIR)/python-cc.mk),$(eval $(e))) \
+		$(foreach e,$(shell cat $(WORK_DIR)/python-cc.mk 2> /dev/null),$(eval $(e))) \
```

Comment:
Shouldn't be echoed somewhere somehow?  Even perhaps in `$(WORK_DIR)/python-cc.mk.error.log`.  Otherwise redirecting it to `/dev/null` will complexity debugging.

Reply:
It is boring me for a long time, that it looks like an error for packages without python-wheels and regularly no python-cc.mk file.
```
===>  Processing wheels of {package-name}
cat: /spksrc/spk/{package-name}/work-{arch}-6.1/python-cc.mk: No such file or directory
```

The only error output that `cat` might produce is `No such file or directory`.


The real problem might be that `$(WHEELS)` is not empty. But anyway this should be moved to a dedicated PR.

Reply:
indeed that is the only side effect but at least we know when it is happening.  Why not simply redirect it to stdout (`2>&1`)?